src/components/themed/TransactionListHeader.js

```diff
@@ -182,6 +190,19 @@ const getStyles = cacheStyles((theme: Theme) => ({
   },
   earnInterestText: {
     fontFamily: theme.fontFaceBold
+  },
+
+  // Transactions Divider
+  transactionsDividerContainer: {
+    flex: 1,
+    marginTop: theme.rem(1),
+    paddingBottom: theme.rem(0.5),
+    borderBottomWidth: theme.rem(1 / 16),
```

Comment:
Using 1/16 makes the assumption that REM equals 16. Please add a parameter to the theme `thinLineWidth` Which does not use REM and is simply 1 display pixel

Reply:
Replace all of your use of 1/16 with thinLineWidth

I noticed you occasionally have a rem 2/16 as a line width. Add another theme parameter `mediumLineWidth` Which would be 2 display pixels

Reply:
fixed